lib/engine/action/bid.rb

```diff
@@ -5,19 +5,21 @@
 module Engine
   module Action
     class Bid < Base
-      attr_reader :company, :corporation, :price
+      attr_reader :company, :corporation, :minor, :price
 
-      def initialize(entity, price:, company: nil, corporation: nil)
+      def initialize(entity, price:, company: nil, corporation: nil, minor: nil)
         @entity = entity
         @company = company
         @corporation = corporation
+        @minor = minor
         @price = price
       end
 
       def self.h_to_args(h, game)
         {
           company: game.company_by_id(h['company']),
           corporation: game.corporation_by_id(h['corporation']),
+          minor: game.minor_by_id(h['minor']),
```

Comment:
in 1846 we use the company card as a minor, does that not work here as well?

Reply:
1846 doesn't process bids for minors directly. Instead it uses a private with the same name as the minor. I'd rather not do that for 18Mag since it has 13 minors and no privates. I can do that if you feel strongly about not adding to action::bid